docs/.eleventy.js

```diff
@@ -39,4 +40,19 @@ module.exports = function(eleventyConfig) {
   // Ignore liquid parsing on these files
   eleventyConfig.ignores.add('email/templates/code');
   eleventyConfig.ignores.add('email/templates/examples');
+
+  eleventyConfig.addTransform("htmlmin", function(content, outputPath) {
+    if (outputPath && outputPath.endsWith(".html")) {
```

Comment:
This is just a personal opinion: I tend to prefer [guard clauses](https://maximegel.medium.com/what-are-guard-clauses-and-how-to-use-them-350c8f1b6fd2) over nested if statements.
The block below could be somenthing like (pseudo)
```
if (!html) {
  return content
}

if (contains 3 dashes thing I did not fully understand) {
  return content
}

return minify(...)
```
Again this is not really an objective comment (feel free to discard).